### PR TITLE
Update basic usage docs for 0.9

### PR DESCRIPTION
**Pull request by sohkai**
In general I've tried to make the language consistent with what the server uses (e.g. "outputs" instead of "conditions").

Note that only https://github.com/bigchaindb/bigchaindb-driver/pull/234/commits/9343ebea186555520e5f5f74334542abd3547faf and https://github.com/bigchaindb/bigchaindb-driver/pull/234/commits/cb83a9d4c9e17e2d37d8eb4c5a861535da2a2b95 contain non-whitespace changes.

-----------

### Preview:

basic usage examples

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -12,11 +12,12 @@ Basic Usage Examples
 
 
 The BigchainDB driver's main purpose is to connect to one or more BigchainDB
-server nodes, in order to perform supported API calls documented under
-:doc:`drivers-clients/http-client-server-api`.
+server nodes, in order to perform supported API calls (see the server's
+`HTTP API docs <https://docs.bigchaindb.com/projects/server/en/latest/drivers-clients/http-client-server-api.html>`_
```
Comment (sohkai) [inline docs/usage.rst]: Not sure if there's a better way to link to this now?

Comment (sbellem) [inline docs/usage.rst]: I think we need to handle https://github.com/bigchaindb/bigchaindb-driver/issues/105

File: docs/usage.rst [hunk 2]
```diff
@@ -199,31 +199,44 @@ or simply use ``fulfilled_creation_tx``:
...
-    In [0]: condition = creation_tx['conditions'][cid]
+    In [0]: transfer_asset = {
+       ...:     'id': asset_id,
+       ...: }
+
+Let's now prepare the transfer transaction:
+
+.. ipython::
+
+    In [0]: fulfilled_output_id = 0
```
Comment (sbellem) [inline docs/usage.rst]: How about calling it `index` instead of `id` to make it clearer, since it is really the `index` of the element of a list.

Comment (sbellem) [inline docs/usage.rst]: i.e.: `fulfilled_output_index`

File: docs/usage.rst [hunk 3]
```diff
@@ -199,31 +199,44 @@ or simply use ``fulfilled_creation_tx``:
...
+
+    In [0]: fulfilled_output = creation_tx['outputs'][fulfilled_output_id]
```
Comment (sbellem) [inline docs/usage.rst]: `output` and `output_index` would be simpler and less confusing as the output has not been fulfilled yet -- we can assume (or hope) that the reader will understand with the help of the context of the example

**Pull request merged**